bigtop-manager-server/src/main/resources/ddl/MySQL-DDL-CREATE.sql

```diff
@@ -58,89 +75,255 @@ CREATE TABLE `user`
     `status`      BIT(1)      DEFAULT 1 COMMENT '0-Disable, 1-Enable',
     `create_time` DATETIME    DEFAULT NULL,
     `update_time` DATETIME    DEFAULT NULL,
+    `create_by`   BIGINT,
+    `update_by`   BIGINT,
     PRIMARY KEY (`id`),
     UNIQUE KEY `uk_username` (`username`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
 
 CREATE TABLE `cluster`
 (
-    `id`           BIGINT(20) UNSIGNED NOT NULL AUTO_INCREMENT,
-    `cluster_name` VARCHAR(255) DEFAULT NULL COMMENT 'Cluster Name',
-    `cluster_desc` VARCHAR(255) DEFAULT NULL COMMENT 'Cluster Name',
-    `cluster_type` SMALLINT UNSIGNED DEFAULT 1 COMMENT '1-Physical Machine, 2-Kubernetes',
-    `selected`     BIT(1)      DEFAULT 1 COMMENT '0-Disable, 1-Enable',
-    `create_time`  DATETIME     DEFAULT NULL,
-    `update_time`  DATETIME     DEFAULT NULL,
+    `id`            BIGINT(20) UNSIGNED NOT NULL AUTO_INCREMENT,
+    `cluster_name`  VARCHAR(255) DEFAULT NULL COMMENT 'Cluster Name',
+    `cluster_desc`  VARCHAR(255) DEFAULT NULL COMMENT 'Cluster Name',
```

Comment:
comment for `cluster_desc` are incorrect

Reply:
Thank you checking, I will add comments